examples/site/testing_pubsub/README.md

```diff
@@ -0,0 +1,115 @@
+# How-to Guide: Testing Event-driven Functions (Pub/Sub triggered)
+
+[buildpacks]: https://buildpacks.io
+[boost-log-gh]: https://github.com/boostorg/log
+[/examples/site/hello_world_pubsub/hello_world_pubsub.cc]: /examples/site/hello_world_pubsub/hello_world_pubsub.cc
+[pubsub_unit_test.cc]: pubsub_unit_test.cc
+[pubsub_integration_server.cc]: pubsub_integration_server.cc
+[pubsub_integration_test.cc]: pubsub_integration_test.cc
+[quickstart-guide]: /google/cloud/functions/quickstart/README.md
+[pubsub-quickstart]: https://cloud.google.com/pubsub/docs/quickstart-console
+
+Event-driven functions do not return values, their only observable behavior are
+side-effects. All tests, therefore, are looking at the side-effects of these
+functions. Depending on where the function is deployed this might be more or
+less involved.
+
+## Function under Test
+
+We will use this function throughput this guide:
+
+[/examples/site/hello_world_pubsub/hello_world_pubsub.cc]
+```cc
+namespace gcf = ::google::cloud::functions;
+
+// Use Boost.Archive to decode Pub/Sub message payload
+std::string decode_base64(std::string const& base64);
+
+void hello_world_pubsub(gcf::CloudEvent event) {  // NOLINT
```

Comment:
Not sure whether `NOLINT` comments are helpful or needed in docs like this. Up to you.

Reply:
Removed, thanks.